### PR TITLE
refactor use of java parameter options

### PR DESCRIPTION
**Pull request by mgudemann**
this tries to follow more closely the coding standard for option parameters

### CODE REVIEW

File: src/java_bytecode/java_parse_options.cpp [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+#include <util/config.h>
+#include <util/message.h>
+#include <util/suffix.h>
+
+#include <json/json_parser.h>
+
+#include "java_parse_options.h"
+
+std::string java_get_cp_include_files(
```
Comment (thk123) [inline src/java_bytecode/java_parse_options.cpp]: Function missing documentation.

File: src/java_bytecode/Makefile [hunk 2]
```diff
@@ -21,6 +21,7 @@ SRC = bytecode_info.cpp \
       java_root_class.cpp \
       java_types.cpp \
       java_utils.cpp \
+      java_parse_options.cpp \
```
Comment (thk123) [inline src/java_bytecode/Makefile]: Should be in alphabetical order (this definitely needs to be added to the githook and tested automatically)

**Comment by mgudemann** [comment]
I will then wait for this to trickle down to `test-gen-support` to add this in the original issue

File: src/java_bytecode/java_parse_options.cpp (hunk 1 above)
Comment (mgudemann) [inline src/java_bytecode/java_parse_options.cpp]: done

File: src/java_bytecode/Makefile (hunk 2 above)
Comment (mgudemann) [inline src/java_bytecode/Makefile]: donedone

**Comment by mgudemann** [comment]
obsoleted by https://github.com/diffblue/cbmc/pull/1221/

**Pull request closed without merging**